docs/tutorial/using-checkbox/test-output.rst

```diff
@@ -0,0 +1,80 @@
+.. _test_output:
+
+============================
+Test output
+============================
+
+When Checkbox runs the tests, it generates a series of outputs, which include the ID and category of the tests, along with the results of each them (if a test ID fails, the cause for the failure would be displayed). 
+
+The test output can be generally divided into the following sections:
+
+- **Hardware Manifest**:
+  
+  Some tests rely on the user's manual confirmation of hardware devices on the machine. This section lists the user's confirmation results.
+
+- **Test ID**:
+  
+  This section contains all the test IDs run during the test. Each test ID has its own separate section indicating the logs and test results during the test.
+
+- **Outcome**:
+  
+  This section indicate the test results of test IDs.
+  
+- **Summary**:
+
+  Graphical icons is used to indicate the results of each test IDs.
+
+- **Test Reports**:
+  
+  Checkbox collects all data related to the test and generates a test report in local.
+
+- **Submit Results**:
+  
+  Users are asked whether they want to upload the results to the Canonical :term:`Certification website`.
+  
+
+Now that you have a basic understanding of the test output, here is an example of audio detect tests for practice:
+
+.. code-block:: none
+
+    =========[ Running job 1 / 3. Estimated time left (at least): 0:00:01 ]=========
```

Comment:
This is not described, progress markers and time estimations are useful, just make the reader notice them